x-pack/platform/test/api_integration/services/fleet_and_agents.ts

```diff
@@ -35,5 +35,86 @@ export async function FleetAndAgentsProvider({ getService }: FtrProviderContext)
         .send({ forceRecreate: true })
         .expect(200);
     },
+
+    async generateAgent(
```

Comment:
copied from `fleet_api_integration/` that is not relocated yet. Other functions related to Fleet & Agent will be migrated to service for easier access

Reply:
cool, can you leave a TODO comment please, it will help us look for it in the future